docker/onePollerPerContainer/docker-compose.tmpl

```diff
@@ -1,23 +1,25 @@
-version: "3.5"
```

Comment:
If i recall kompose (k8 artifacts) were not generated without 3+ version. pls do check once.

Reply:
Yeah it's a bit of a [mess](https://docs.docker.com/compose/compose-file/compose-versioning/#versioning). Sticking with `3.5` seems reasonable.  The reason I wanted to remove it doesn't exist anymore so I'll add it back